com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -2237,6 +2239,12 @@ void InitializeGlobalResources(ScriptableRenderContext renderContext)
             else
                 FrameSettings.AggregateFrameSettings(ref currentFrameSettings, camera, additionalCameraData, m_Asset);
 
+            // Sanity check.
+            if (xrPass.enabled && currentFrameSettings.IsEnabled(FrameSettingsField.AsymmetricProjection))
+            {
+                Debug.LogError("XR and explicit Asymmetric Projection (Frame Settings) are not expected to be enabled simultaneously.");
```

Comment:
dont do this, this is hard to catch and polute the log. Instead disable (i.e grey in the ux) the framesettings if XR is enabled, and in the agreggate framesettings function do something like this:
FrameSettingsField.AsymmetricProjection = FrameSettingsField.AsymmetricProjection && XREnabled
We do that for many other settings, you can take example on it.

Reply:
Actually I can just remove this, it is a somewhat overly defensive piece of code I just added since using XR and Cluster Display at the same time doesn't really make sense. But it doesn't actually break anything afiak. I'll add a check in our Cluster Display package instead.